apps/staking/src/config/server.ts

```diff
@@ -74,7 +74,9 @@ export const GOVERNANCE_ONLY_REGIONS = transformOr(
   [],
 );
 export const PROXYCHECK_API_KEY = demandInProduction("PROXYCHECK_API_KEY");
-
+// This needs to be a public key that has SOL in it all the time, it will be used as a payer in the transaction simulation to compute the claimable rewards
+// such simulation fails when the payer has no funds.
+export const SIMULATION_PAYER = process.env.SIMULATION_PAYER;
```

Comment:
I doesn't seem to be working in the vercel preview and I can't quite figure out why

Reply:
What's failing exactly?  Is the expected key not being used?

Also, is this key safe to reveal publicly and hardcode into the codebase?  I assume so since it will be serialized to the browser and it's a public key; if so it's better to do this as `getOr("SIMULATION_PAYER", "<default value>")` and leave it unset in Vercel; ideally we would keep the environment as minimal as possible and only set things when we need a quick override or for secrets

Reply:
sure